extensions/amp-user-notification/0.1/amp-user-notification.js

```diff
@@ -257,7 +261,23 @@ export class AmpUserNotification extends AMP.BaseElement {
 
   /** @override */
   show() {
-    this.element.style.display = '';
+    const viewerPaddingTop = this.viewer_.getPaddingTop();
+
+    // if there is no viewer padding, then no need to do any offset
+    if (viewerPaddingTop > 0) {
+      // we need to use getComputedStyle in cases where `top` is not in pixels
+      const currentTop = parseFloat(this.win_
+          ./*REVIEW*/getComputedStyle(this.element, null)
+          .getPropertyValue('top'));
+
+      const top = Math.max(viewerPaddingTop, currentTop);
```

Comment:
This assumes that `top` is defined in `px`. Also, I believe it should be `paddingTop + top`, no? Please confirm. IN the meantime, I'm gonna go ahead and try to do this via `FixedLayer` - so please don't rush fixing. Just let me know how you think it should work.